src/pipeline/node/AutoCalibration.cpp

```
@@ -10,6 +11,36 @@
 namespace dai {
 namespace node {
 
+namespace {
+
+bool hasDifferentDistortion(const CalibrationHandler& lhs, const CalibrationHandler& rhs, CameraBoardSocket socket) {
```

Cppcheck (2.20.0)

[information] Include file

(missingIncludeSystem)